src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -2323,12 +2323,12 @@ private void processFileChanges(
                     enableServerDebug(false);
                 }
                 if (container && System.getProperty("os.name").equalsIgnoreCase("linux")) {
-                    info("Restarting the container is required on Linux for it to take effect.");
+                    info("Now restarting the container for this change to take effect which is required on Linux.");
```

Comment:
The message still seems a bit verbose.  Does the user care on what OS the restart is required?  If they are on Linux then they should not be concerned about other OSes.  Perhaps we should just say "Restarting the container for this change to take effect..."

Reply:
Ok.